### ISSUE TITLE
Feature Request: Filter polyX

### ISSUE THREAD

**Issue by babytut** [OpenGene/fastp#268]
We use NovaSeq to sequencing and have polyG problem. Now we want to remove all reads in 2 paired-end files which has polyG length larger than such as 50 bases.

**Comment by sfchen** [comment]
PolyG removing is enabled by default for NovaSeq data, have you try it?

**Comment by babytut** [comment]
No, I want to remove all reads which have PolyG lengh larger than  50 bases, not trimming like here [polyg tail trimming](https://github.com/uc-cdis)

**Comment by alsmnn** [comment]
+ 1 for filtering of whole read instead of trimming.
